PhpcsChanged/Cli.php

```diff
@@ -257,6 +258,7 @@ function runGitWorkflow(array $gitFiles, array $options, ShellOperator $shell, c
 		throw $err; // Just in case we do not actually exit
 	}
 
+
```

Comment:
Minor: but this feels like an accidental addition?

Reply:
Correct. Addressed